webpack.config.js

```diff
@@ -12,15 +12,6 @@ module.exports = {
         exclude: /node_modules/,
         use: ['babel-loader'],
       },
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        loader: 'eslint-loader',
-        options: {
-          // allow errors to not stop compiling
-          emitWarning: true,
-        },
-      },
```

Comment:
I had a feeling it was conflicting but let me know if we still want this in webpack config

Reply:
I think they are in different areas. this should be safe to keep.

Reply:
Added it back here